### PR TITLE
pipeline: add results check

### PR DESCRIPTION
**Pull request by FlorisFeddema**

### CODE REVIEW

File: charts/home-assistant/values.yaml [hunk 1]
```diff
@@ -41,4 +41,4 @@ config:
     allowPrivilegeEscalation: false
     capabilities:
       drop: [ "ALL" ]
-      add: []
+#      add: []
```
Comment (rouke-broersma) [inline charts/home-assistant/values.yaml:44]: Add should be explicitly empty no?

File: charts/home-assistant/values.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ image:
   tag: 2024.8.3
 
 nodeName:
-tolerations: []
+#tolerations: []
```
Comment (rouke-broersma) [inline charts/home-assistant/values.yaml:7]: Does empty not suffice?

Comment (FlorisFeddema) [reply charts/home-assistant/values.yaml:7]: Oh yes it should

File: charts/home-assistant/values.yaml (hunk 1 above)
Comment (FlorisFeddema) [reply charts/home-assistant/values.yaml:44]: https://github.com/FlorisFeddema/homelab/pull/1277 shows it is a problem

Comment (rouke-broersma) [reply charts/home-assistant/values.yaml]: Not empty array, empty prop

Comment (FlorisFeddema) [reply charts/home-assistant/values.yaml]: https://github.com/FlorisFeddema/homelab/pull/1281, is now fixed 👍🏼

**Pull request merged**